static/js/components/SearchResult.js

```diff
@@ -27,6 +27,7 @@ const PostSearchResult = ({ post, toggleUpvote }: PostProps) => (
     post={post}
     isModerator={false}
     menuOpen={false}
+    menuDisabled={true}
     toggleUpvote={toggleUpvote}
```

Comment:
this is kind of just syntax sugar, but if you want to pass a value of `true` for a boolean prop (like with `menuDisabled` here) you can just do this:

```js
<Component booleanProp />
```

instead of

```js
<Component booleanProp={true} />
```

Reply:
✔️